src/main/deploy/auton/BlueAmpWing1Center0.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE auton SYSTEM "auton.dtd">
+<auton>
+
+    <primitive id = "RESET_POSITION_PATH_PLANNER"
+                pathname= "BlueAmpWing1Center0"/>
+    
+    <snippet file = "ShootWhenStop.xml"/>
+
+    <primitive id = "DRIVE_PATH_PLANNER"
+                pathname = "BlueAMpWing1Center0"/>
```

Comment:
add zones for picking up the second note ... example


    <primitive id= "DRIVE_PATH_PLANNER"
            pathname="BlueSubwooferWing1Center0">
            <zone filename="BlueIntakeWingNote2.xml"/>
    </primitive>